docs: added example in docs of using a resolver for metatags for markdown component (#1014) (#1082)
analogjs/analog: apps/docs-app/docs/features/routing/content.md

}
```

### Using A Resolver For Metatags

> The `postMetaResolver` function is responsible for fetching the meta tags for a post. This function should return an array of meta tags. Here's an example:

Reviewer: ```suggestion
In your route configuration, you can use the `RouteMeta` object to resolve meta tags for a route. This is done by assigning the `postMetaResolver` function to the `meta` property. 

Below is an example of using a `postMetaResolver` function that fetches the meta tags for a post. This function returns an array of meta tags.
```

```ts
export const postMetaResolver: ResolveFn<MetaTag[]> = (route) => {
  const postAttributes = injectActivePostAttributes(route);

  return [
    {
      name: 'description',
      content: postAttributes.description,
    },
    {
      name: 'author',
      content: 'Analog Team',
    },
    {
      property: 'og:title',
      content: postAttributes.title,
    },
    {
      property: 'og:description',
      content: postAttributes.description,
    },
    {
      property: 'og:image',
      content: postAttributes.coverImage,
    },
  ];
};
```

In your route configuration, you can use the `RouteMeta` object to resolve meta tags for a route. This is done by assigning the `postMetaResolver` function to the `meta` property. Here's an example:

```ts
export const routeMeta: RouteMeta = {
  title: postTitleResolver,
  meta: postMetaResolver,
};
```

The resolved meta tags can be accessed in the `BlogPostComponent` using the `ActivatedRoute` service. These meta tags can then be passed to the `MarkdownComponent`. Here's an example:

```ts
export default class BlogPostComponent {
  readonly route = inject(ActivatedRoute);

>   readonly metaTags$ = this.route.data.pipe(pluck('meta'));

Reviewer: Isn't `pluck` deprecated in RxJS?

```suggestion
  readonly metaTags$ = this.route.data.pipe(map(data => data['meta']));
```

Author: Yup as of RxJS 7 it is

  // In the template

>   <analog-markdown [content]="post.content" [metaTags]="metaTags$ | async"></analog-markdown>

Reviewer: ```suggestion
  <my-component [metaTags]="metaTags$ | async"></my-component>
```

}
```

> The `MarkdownComponent` should be modified to accept an input for meta tags and use these when rendering the markdown content. Here's an example:

Reviewer: I think this section should be removed

Author: Removed as requested

withMarkdownRenderer({
  loadMermaid: () => import('mermaid'),
});

> ````

Reviewer: ```suggestion
```
```
